### PR TITLE
npm release workflow

### PR DESCRIPTION
**Pull request by Zacharis278**
changed filename to `release` since it looks like other repos have been using that name

### CODE REVIEW

**Comment by Zacharis278** [comment]
@TamoshaytisV edit a dry run of the release into the CI build. Seems to have passed. https://github.com/edx/frontend-lib-special-exams/pull/3/checks?check_run_id=2578711768

Anything you think we're missing?

**Comment by TamoshaytisV** [comment]
@Zacharis278 not sure if at this point we will have `dist` artifacts. have not found anything about it in dry-mode logs
All other stuff looks good to me so i suggest to try it out

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Install dependencies
+        run: npm ci
+      - name: Test
+        run: npm run test
+      - name: Release
```
Comment (TamoshaytisV) [inline .github/workflows/release.yml:21]: Do we need to have also ?
```yml
- name: Build
    run: npm run build
```

Comment (Zacharis278) [reply .github/workflows/release.yml:21]: yeah we did, I edit a published hook that I think should do it

**Review by TamoshaytisV — APPROVED** [review]

**Review by idegtiarov — APPROVED** [review]
LGTM so far!

**Pull request merged**